packages/drivers/routerlicious-urlResolver/src/urlResolver.ts

```diff
@@ -53,6 +53,7 @@ export class RouterliciousUrlResolver implements IUrlResolver {
         const path = reqUrl.pathname.split("/");
         let tenantId: string;
         let documentId: string;
+        let relativePath: string;
```

Comment:
`relativePath` is potentially `undefined` when used later, which will append the word `"undefined"` to the result.

I think this could just be:

`const relativePath = path.slice(4).join("/")`